tests/controls/python_spec.rb

```diff
@@ -0,0 +1,39 @@
+title 'Python & Libraries'
+
+control 'Python version' do
+  impact 'high'
+  title 'Python should be a specific version'
+  desc 'Python should be a specific version'
```

Comment:
It's good to guard against changing python version by mistake, but do we care about it being 3.7.1 specifically?

Reply:
Yeah I think we can relax it to a minor version rather than a patch. Saying that, it's been 3.7.1 for the last set of images, and 3.8 is likely to be pulled down soon. I'm happy to be guided on this one

Reply:
Maybe we don't need this test, if no-one is too fussed about the specific version?

Reply:
I think we should at least be notified, as when we do change Python version it can break things in nasty ways.